app/views/posts/index.html.erb

```diff
@@ -1,3 +1,6 @@
+
+<h3 Welcome you are successfully signed up </h3>
```

Comment:
I never see this message on the posts page

Reply:
you will now

Reply:
Confirm that I can see this